Samples/BatchSupport/BatchSupport/BatchSupport.vcxproj

```diff
@@ -91,20 +93,21 @@
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <IntrinsicFunctions>true</IntrinsicFunctions>
       <PreprocessorDefinitions>NDEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories Condition="'$(Configuration)|$(Platform)'=='Release|x64'">..\..\SampleSharedLib\SampleSharedLib;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
You don't need the Configuration == Release condition because of the `ItemDefinitionGroup` it's in (line 90), right?

Reply:
This is created by VS. What I did was applied the includeDirectiores to both Release and Debug.

---
In reply to: [401119857](https://github.com/microsoft/Windows-Machine-Learning/pull/305#discussion_r401119857) [](ancestors = 401119857)